### PR TITLE
8.0 : Add module hr_expense_sequence

### PR DESCRIPTION
**Pull request by alexis-via**
This PR is the same as the following PR for v7 https://github.com/OCA/hr/pull/42
Please comment on the PR for v7.

### CODE REVIEW

File: hr_expense_sequence/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+##############################################################################
+
+{
+    'name': 'Sequence on Expenses',
+    'version': '0.1',
+    'category': 'Human Resources',
+    'license': 'AGPL-3',
+    'summary': "Adds a sequence on expenses",
+    'description': """
+Sequence on Expenses
```
Comment (pedrobaeza) [inline hr_expense_sequence/__openerp__.py:30]: You can put this as a separate file README.rst

File: hr_expense_sequence/hr_expense_sequence.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+
+class hr_expense_expense(orm.Model):
+    _inherit = 'hr.expense.expense'
+    _order = 'name desc'
+
+    _columns = {
+        # Move the description from the 'name' field to 'description' field
+        # In the 'name' field, we now store the number/sequence
+        'name': fields.char('Number', size=32, readonly=True),
```
Comment (pedrobaeza) [inline hr_expense_sequence/hr_expense_sequence.py]: Please use new API to only overwrite changed attribute (string).

**Comment by pedrobaeza** [comment]
Two minor issues, but the rest LGTM.

**Comment by alexis-via** [comment]
OK pedro, let's go for the new API on this small module, I know you like it :)

**Comment by pedrobaeza** [comment]
Cool! :smile:

**Comment by pedrobaeza** [comment]
:+1:

**Comment by max3903** [comment]
Module has already been merged with #70

**Comment by pedrobaeza** [comment]
Sorry for overlapping, but I even didn't remember this PR when I had again the need. Do you want credits in my module, @alexis-via?

**Comment by alexis-via** [comment]
Arf, incredible !

@pedrobaeza Don't worry about that. It's your code, not mine, so you don't need to give me any credits.

**Comment by alexis-via** [comment]
@pedrobaeza 
I had a look at your code in "your" hr_expense_sequence module : I see that you defined a new field "number" to store the sequence. In "my" hr_expense_sequence, I defined a new field "description" to store the description and I used the existing field "name" to store the sequence. Why ? Because the "name" field is copied on the account.move of the expense in the "ref" field, so you can see on the account.move to number of the related expense note, which is quite important for me.

But we could enhance "your" module to do that, via an inherit of account_move_get() (addons/hr_expense.py line 148). What you do think ? If you agree, I'll try to make a PR for that.

**Comment by pedrobaeza** [comment]
@alexis-via, I always prefer not to change data model swapping fields content, specially if no proper installation/uninstallation scripts are providden. Even that, it's always trickier to do that. That's why I added a new column for the sequence.

I'm OK with adding that overriding for change the name of the account move line.

Thanks.

**Comment by alexis-via** [comment]
@pedrobaeza Here is my PR to add the "number" of the expense on the "ref" field of the account.move (that only field I can inherit easily) : https://github.com/OCA/hr/pull/87

**Pull request closed without merging**